src/test/java/TaskTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskTest {
+    @Test
+    public void toDoCreationTest(){
```

Comment:
Perhaps following the `featureUnderTest_testScenario_expectedBehavior()` test method convention will be better.

Reply:
agree with the test method convention:)